coderefs/coderefs.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 func Run(opts options.Options) {
-	dir := opts.Dir
+	dir, _ := validation.NormalizeAndValidatePath(opts.Dir)
 	absPath, err := validation.NormalizeAndValidatePath(dir)
```

Comment:
I was on the fence about that, I did think of updating it inside of `options.go` https://github.com/launchdarkly/ld-find-code-refs/pull/189/commits/c528266e357f9812d46288bed34b80cdfa3f41e2 but that ends up warning about ineffectual assignment(though I think the warning might be wrong).

The problem is `options` struct gets passed around otherwise and I'm not a big fan of mutating `options.Dir` in place though we could do it, another thing I thought about is adding a `ValidatedDir` to the options struct and using that instead throughout the app.